webview/src/plots/components/ZoomablePlot.tsx

```diff
@@ -71,49 +47,45 @@ export const ZoomablePlot: React.FC<ZoomablePlotProps> = ({
   }
 
   return (
-    <ZoomablePlotWrapper spec={plotProps.spec as SpecWithTitles}>
```

Comment:
Are we getting rid of the tooltip?

Reply:
Got my answer a few files below (https://github.com/iterative/vscode-dvc/pull/4734/files#r1405504743)